saleor/graphql/product/dataloaders/products.py

```diff
@@ -199,7 +199,9 @@ def batch_load(self, keys):
     def get_variants_filter(self, products_ids, channel_slugs):
         return {
             "product_id__in": products_ids,
-            "channel_listings__channel__slug__in": channel_slugs,
+            "channel_listings__channel__slug__in": [
+                str(slug) for slug in channel_slugs
```

Comment:
Could you explain why we need to do `str` on channel_slugs?

Reply:
channel_slugs in this case is LazyObject, if we pass it directly to django ORM filter, it will break